spec/support/authentication_helper.rb

```diff
@@ -0,0 +1,11 @@
+module AuthenticationHelper
+  def sign_in_as(user)
+    visit root_path
+    click_on 'Sign In'
+
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+
+    click_on 'Log in'
+  end
```

Comment:
Yeah we thought, this would be a great time to add in some code that we wrote ourselves and didn't copy from somewhere else, so there it is!